test/e2e/setup.go

```diff
@@ -124,8 +121,11 @@ func setupInternal(
 	tables = append(tables, bigtable.NewTable(branchTableName, branchTable))
 
 	if useImportGroup {
-		for _, t := range tableConfig.Tables {
+		for _, t := range tableNames {
 			name := strings.TrimSpace(t)
```

Comment:
nit: maybe we can safely trust the tableNames returned by util.ParseBigtableGroup() if it also skips empties? Especially since we are doing it in two places...

Reply:
Removed the extra check